models/models.js

```diff
@@ -0,0 +1,5 @@
+const db = require('../db/connection');
+
+exports.retrieveTopics = () => {
+    return db.query("SELECT * FROM topics;");
```

Comment:
I'd maybe get the array here and send it back, instead of the response, with rows etc. 

Its up to you, but you'll probably find it easier on the front end if you can just access the array via the key of topics, instead of topics.rows.